po/fr.po

```diff
@@ -0,0 +1,170 @@
+# French translations for com.github.sgpthomas.hourglass package.
+# Copyright (C) 2020 THE com.github.sgpthomas.hourglass'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.sgpthomas.hourglass package.
+# Nathan Bonnemains (@NathanBnm), 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.sgpthomas.hourglass\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-05-03 12:04+0200\n"
+"PO-Revision-Date: 2020-05-03 11:55+0200\n"
+"Last-Translator: Nathan Bonnemains (@NathanBnm)\n"
+"Language-Team: none\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: src/Dialogs/MultiSelectPopover.vala:32
+#: src/Dialogs/MultiSelectPopover.vala:83
+msgid "Sun"
+msgstr "Dim"
+
+#: src/Dialogs/MultiSelectPopover.vala:32
+#: src/Dialogs/MultiSelectPopover.vala:83
+msgid "Mon"
+msgstr "Lun"
+
+#: src/Dialogs/MultiSelectPopover.vala:32
+#: src/Dialogs/MultiSelectPopover.vala:83
+msgid "Tue"
+msgstr "Mar"
+
+#: src/Dialogs/MultiSelectPopover.vala:32
+#: src/Dialogs/MultiSelectPopover.vala:83
+msgid "Wed"
+msgstr "Mer"
+
+#: src/Dialogs/MultiSelectPopover.vala:32
+#: src/Dialogs/MultiSelectPopover.vala:83
+msgid "Thu"
+msgstr "Jeu"
+
+#: src/Dialogs/MultiSelectPopover.vala:32
+#: src/Dialogs/MultiSelectPopover.vala:83
+msgid "Fri"
+msgstr "Ven"
+
+#: src/Dialogs/MultiSelectPopover.vala:32
+#: src/Dialogs/MultiSelectPopover.vala:83
+msgid "Sat"
+msgstr "Sam"
+
+#: src/Dialogs/MultiSelectPopover.vala:88
+msgid "Every Day"
+msgstr "Chaque jour"
+
+#: src/Dialogs/MultiSelectPopover.vala:90
+msgid "Every Weekday"
+msgstr "Chaque jour de la semaine"
+
+#: src/Dialogs/MultiSelectPopover.vala:92
+msgid "Every Weekend"
+msgstr "Chaque weekend"
+
+#: src/Dialogs/MultiSelectPopover.vala:99 src/Dialogs/NewAlarmDialog.vala:100
+msgid "Never"
+msgstr "Jamais"
+
+#: src/Dialogs/NewAlarmDialog.vala:120 src/Widgets/AlarmTimeWidget.vala:109
+msgid "Edit"
+msgstr "Modifier"
+
+#: src/Dialogs/NewAlarmDialog.vala:120
+msgid "Create"
+msgstr "Créer"
+
+#: src/Dialogs/NewAlarmDialog.vala:122
+msgid "Cancel"
+msgstr "Annuler"
+
+#: src/Dialogs/NewAlarmDialog.vala:124
+msgid "Delete"
+msgstr "Supprimer"
+
+#: src/Dialogs/NewAlarmDialog.vala:143
+msgid "Title:"
+msgstr "Nom :"
+
+#: src/Dialogs/NewAlarmDialog.vala:148
+msgid "Choose a time:"
+msgstr "Sélectionner une heure :"
+
+#: src/Dialogs/NewAlarmDialog.vala:153
+msgid "Choose a date:"
+msgstr "Sélectionner une date :"
+
+#: src/Dialogs/NewAlarmDialog.vala:163
+msgid "Repeat:"
+msgstr "Répétition :"
+
+#: src/Widgets/Alarm.vala:129
+msgid "Repeats "
+msgstr ""
+
+#: src/Widgets/Alarm.vala:129
+msgid ", Repeats "
+msgstr ""
```

Comment:
As long as I can't use the app correctly because of #71 I can't manage to see what does these strings refers to? Any additional context?

Reply:
![Screenshot from 2020-04-28 19-42-42](https://user-images.githubusercontent.com/26003928/80912230-782b9180-8d76-11ea-9bb9-51e4aaad9ae8.png)

This seems to be a string shown in the timer list, and when a set date of a timer is a different month from now, this string is used like: `5/3, Repeats Sun Mon`